app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -13,10 +13,13 @@
 import com.bumptech.glide.request.RequestOptions;
 import com.bumptech.glide.request.target.Target;
 import org.oppia.android.R;
+import org.oppia.android.app.model.ChapterPlayState;
 import org.oppia.android.app.model.LessonThumbnailGraphic;
 import org.oppia.android.app.model.ProfileAvatar;
 
-/** Holds all custom binding adapters that bind to [ImageView]. */
```

Comment:
This formatting is correct: https://developer.android.com/kotlin/style-guide?authuser=1#formatting_2 and generally what I've been advising people to use to help make code more concise when possible.

Reply:
Reverted to previous format.